pkg/regexputil/opt.go

```diff
@@ -120,3 +120,64 @@ func expandRunes(s []rune) [][]rune {
 	}
 	return ret
 }
+
+// Wildcard attempts to convert the regexp into a wildcard form, that is, if it
+// can be represented as a glob pattern using only "*", it will return "string*"
+// and true. It is assumed the "*" glob character matches 0 or more characters
```

Comment:
Have tested that assumption by adding (locally) the following 2 tests to [TestTagVWildcardFilter.java](https://github.com/OpenTSDB/opentsdb/blob/master/test/query/filter/TestTagVWildcardFilter.java):
```
  @Test
  public void left() throws Exception {
    TagVFilter filter = new TagVWildcardFilter(TAGK, "*ogg-01.ops.ankh.morpork.com");
    assertTrue(filter.match(tags).join());
  }
  
  @Test
  public void right() throws Exception {
    TagVFilter filter = new TagVWildcardFilter(TAGK, "ogg-01.ops.ankh.morpork.com*");
    assertTrue(filter.match(tags).join());
  }
```